docs/install/install-serving-with-yaml.md

```diff
@@ -34,6 +34,15 @@ To install the serving component:
 
 ## Install a networking layer
 
+The networking layer in Knative Serving routes incoming and outgoing traffic for your Knative installation.
```

Comment:
I'd sort of prefer to refer to this as "HTTP router" or "HTTP ingress" (it's more precise, and avoids confusion with e.g. Kubernetes CNI layer).

The networking layer in question here is responsible for incoming requests and the associated responses. It doesn't do anything for outbound traffic from the user containers.

Reply:
Thanks @evankanderson for your suggestions. Would it make sense for me to change the wording to something like:

> The networking layer in Knative Serving is responsible for incoming requests and the associated responses for your Knative installation.